ui/src/app/shared/components/deep-links.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import {LinkInfo} from '../models';
+
+export const DeepLinks = (props: {links: LinkInfo[]}) => {
+    const {links} = props;
+    return (
+        <div style={{margin: '10px 0'}}>
+            {(links || []).map((link: LinkInfo) => (
+                <div key={link.title} style={{display: 'flex', alignItems: 'center', height: '35px'}}>
+                    <a href={link.url} target='_blank' style={{display: 'flex', alignItems: 'center', marginRight: '7px'}} rel='noopener'>
```

Comment:
I know these links come from the admin-controlled ConfigMap, but all the same can we validate that the href isn't a script?

Some work can probably be re-used from here: https://github.com/argoproj/argo-cd/commit/8bc3ef690de29c68a36f473908774346a44d4038